### PR TITLE
Add new sample for js 

### PR DESCRIPTION
**Pull request by goldenlimit**
Using localStorage to dynamically center the map based on last infoWindow location

### CODE REVIEW

**Comment by jgravois** [comment]
looks like you have several merge commits in here.  not a huge issue, but a good opportunity to try [squashing](http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request) your work down into a single commit.

File: web-js/infoWindow-location-save-in-localStorage/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Resources
+
+* [ArcGIS for JavaScript API Resource Center](http://help.arcgis.com/en/webapi/javascript/arcgis/index.html)
+* [Local storage - experimental](https://developers.arcgis.com/javascript/jssamples/exp_localstorage.html)
+* [ArcGIS Blog](http://blogs.esri.com/esri/arcgis/)
+* [twitter@esri](http://twitter.com/esri)
+
+## Issues
+
```
Comment (jgravois) [inline web-js/infoWindow-location-save-in-localStorage/README.md]: you dont need an **Issues** and **Contributing** section for the readme for a single sample.

that being said, you *should* add a link to make it convenient for people to view the live sample once the new sample is merged into `master` (and afterward `gh-pages`)

```
[Live Sample](http://esri.github.io/developer-support/web-js/[folder-name]/index.html)
```

**Comment by goldenlimit** [comment]
@jgravois Thanks for the info. I have a live sample link that point to my personal github page. I am not really sure could I change anything from gh-pages, or I just need to point to that url first and last once the request is merged, it will automatically have?

**Comment by jgravois** [comment]
yup, include the url that it will have after its merged into `master` (and then `gh-pages`)

**Comment by goldenlimit** [comment]
image @jgravois 

I think I did the squash, but I didn't see any changes here in pull request. Am I missing sth? :ear:

**Comment by NoashX** [comment]
@goldenlimit @jgravois this looks like a good use case for SDK WAM tomorrow re: squashing and rebasing (?)

**Comment by jgravois** [comment]
@goldenlimit typically people rebase against the `upstream` remote, not `origin` (their own fork).  either way, afterward you need to `git push **origin** [branchname] --force` in order for the changes to be reflected in the existing pull request.

> note: force pushing to your own fork **is** okay when you know what you're doing, why you're doing it, and you are 100% certain that the branch that youre rewriting history for is **not** tracked by anyone else.

**Comment by goldenlimit** [comment]
@jgravois looks like I fix it now. Sorry @NoashX I am OCD on this issue.

**Comment by jgravois** [comment]
nothing wrong with being OCD, but in the future it'd be safe to wait to squash until **after** you've laid down all the extra commits end up being needed during code review

**Comment by goldenlimit** [comment]
Could anyone take a little bit time to test and merge this sample? It being sitting here for a while.

**Comment by NoashX** [comment]
yeah man, works great, thanks for bumping this @goldenlimit

**Pull request merged**